File: app/src/main/java/com/paradigmadigital/repository/settings/NotificationPreferenceFragment.java
package com.paradigmadigital.repository.settings;

import com.paradigmadigital.R;

import android.os.Bundle;

public class NotificationPreferenceFragment extends SettingsFragment {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.pref_notification);
        setHasOptionsMenu(true);
    }
}

File: app/src/main/java/com/paradigmadigital/repository/preferences/NotificationPreferenceFragment.kt
package com.paradigmadigital.repository.preferences

import android.os.Bundle
import com.paradigmadigital.R

class NotificationPreferenceFragment : SettingsFragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.pref_notification)
        setHasOptionsMenu(true)
    }
}
